### ISSUE TITLE
Radio and checkbox group components text label is not showing when using the sub-component syntax

### ISSUE THREAD

**Issue by roelVerdonschot** [bootstrap-vue/bootstrap-vue-next#941]
**Describe the bug.**
Radio and checkbox group components text label is not showing when using the sub-component syntax see your own docs:
https://bootstrap-vue.github.io/bootstrap-vue-next/components/FormRadio.html#grouped-radios

**Comment by roelVerdonschot** [comment]
Also putting a div or any other html element in the group brakes the check-/radio group element and nothing will render on screen. 

```
<b-form-checkbox-group v-model="selected">
                    <div>
                        <b-form-checkbox :value="1">
                            test 123
                        </b-form-checkbox>
                    </div>
                </b-form-checkbox-group>
```

**Comment by VividLemon** [comment]
> Also putting a div or any other html element in the group brakes the check-/radio group element and nothing will render on screen.
> 
> ```
> <b-form-checkbox-group v-model="selected">
>                     <div>
>                         <b-form-checkbox :value="1">
>                             test 123
>                         </b-form-checkbox>
>                     </div>
>                 </b-form-checkbox-group>
> ```

This isn't really a bug. Grouping only checks direct children. It's possible this limitation could be removed, but I'd have to deep dive into how it's functioning

**Comment by VividLemon** [comment]
The second issue is planned to be fixed in 0.8, however, it should probably be noted that Bootstrap-vue likely suffers from the same limitation. In BVN, the limitation is going to be removed, but it is somewhat less straightforward to implement than the typical feature. 

As it pertains to the original issue, this is a duplicate of https://github.com/bootstrap-vue/bootstrap-vue-next/issues/823 , and I will proceed to watch for the outcome of 0.8, to see if the entire issue is solved with the changes made in that release.
